examples/autoinstrument/build.gradle

```diff
@@ -51,6 +51,8 @@ jib {
 	from.image = 'gcr.io/distroless/java-debian10:11'
 	containerizingMode = 'packaged'
 	container.ports = ["8080"]
+	// required for resource detection in GKE environment
```

Comment:
This is a sensible default value for `CONTAINER_NAME`.

However, `NAMESPACE` should not be set in the *image*, it should be passed in as part of the *container* definition when needed. This is for two reasons:

1. It's specific to K8s, but a container image can be used in non-K8s environments (such as Cloud Run or a VM with a container runtime).
2. If someone uses this image in K8s but forgets to set `NAMESPACE`, they'll get a misleading result rather than an error.

If we really need to set a value for `NAMESPACE`, we should use something like `unspecified` to signal that someone needs to modify it.